src/query/storage/m3/dynamic_cluster_test.go

```diff
@@ -75,6 +75,46 @@ func newNamespaceOptions() namespace.Options {
 	return namespace.NewOptions().SetStagingState(state)
 }
 
+func TestDynamicClustersUninitialized(t *testing.T) {
+	t.Parallel()
```

Comment:
Is this necessary?

Reply:
new linting rule, `paralleltest`, that was biting me. Not sure if others have a workaround?

Reply:
I had a workaround here: https://github.com/m3db/m3/pull/2949/commits/d2a85bdb7d4761be0c7fedddc8de0bb7f6f88b34 😜 

Merging master should do it :+1: